### PR TITLE
Various: Merge geometries of capital locality records

### LINKED ISSUE #711: Implode capital locality records
Similar to what happened with Wellington (tracked [here](https://github.com/whosonfirst-data/whosonfirst-data/issues/118)), the following localites have been split into multiple records:

- Athens:

  - https://whosonfirst.mapzen.com/spelunker/id/101752329 <- this geom for Athens is correct. No edits needed. 
  - https://whosonfirst.mapzen.com/spelunker/id/101913513 <- this is actually the rough geom for 1126028555.
  - https://whosonfirst.mapzen.com/spelunker/id/101913511 <- this is actually the rough geom for 1125835375.

- Jerusalem:

  - https://whosonfirst.mapzen.com/spelunker/id/101922189
  - https://whosonfirst.mapzen.com/spelunker/id/101922289
  - https://whosonfirst.mapzen.com/spelunker/id/101922269
  - https://whosonfirst.mapzen.com/spelunker/id/101922261
  - https://whosonfirst.mapzen.com/spelunker/id/101922271
  - https://whosonfirst.mapzen.com/spelunker/id/101922243
  - https://whosonfirst.mapzen.com/spelunker/id/101922257
  - https://whosonfirst.mapzen.com/spelunker/id/101922193

The best thing to do here is to merge all of these into a single record. More work is needed to untangle the areas around Jerusalem.

- Brussels:

  - https://whosonfirst.mapzen.com/spelunker/id/101751573/
  - https://whosonfirst.mapzen.com/spelunker/id/101913231/
  - https://whosonfirst.mapzen.com/spelunker/id/101913235/
  - https://whosonfirst.mapzen.com/spelunker/id/101913241/
  - https://whosonfirst.mapzen.com/spelunker/id/101913243/
  - https://whosonfirst.mapzen.com/spelunker/id/101913245/
  - https://whosonfirst.mapzen.com/spelunker/id/101913247/
  - https://whosonfirst.mapzen.com/spelunker/id/101913233/
  - https://whosonfirst.mapzen.com/spelunker/id/101913239/

Suggest deprecating all, aside from 101751573, then finding new data for the surrounding area. The existing geometries surrounding 101751573 are not Brussels - they are a mix of bad geometries and geometries that need to be split apart.

Related:

- Wellington:

  - See: https://github.com/whosonfirst-data/whosonfirst-data/issues/118

- Bratislava:

  - See: https://github.com/whosonfirst-data/whosonfirst-data/issues/575

### PR DESCRIPTION
**Pull request by stepps00**
Related to #711.

Athens, Brussels, and Jerusalem each have multiple records in Who's On First. This PR takes care of each locality as follows:

**Jerusalem:**
- Updated the geometry and properties of one record
- One record supersedes the rest, with `wof:supersedes` and `wof:superseded_by` links

**Athens:**
- Updated two qs_pg point records with new polygons from bunk Athens records.
- Saved qs_pg points as alt geometries

**Brussels:**
- Updated names, min_zoom, and ne properties in a single Brussels record
- Supersedes bunk records into the single Brussels record, with `wof:supersedes` and `wof:superseded_by` links

### CODE REVIEW

File: data/101/922/189/101922189.geojson [hunk 1]
```diff
@@ -4,18 +4,18 @@
...
-    "geom:area_square_m":2281148.665073,
-    "geom:bbox":"35.224571228,31.7725906372,35.2400054932,31.7972145081",
-    "geom:latitude":31.784227,
-    "geom:longitude":35.231835,
+    "geom:area":0.000265,
+    "geom:area_square_m":2790441.745664,
+    "geom:bbox":"35.224571228,31.7391853333,35.2525596619,31.7972145081",
+    "geom:latitude":31.777791,
+    "geom:longitude":35.234177,
     "iso:country":"PS",
```
Comment (nvkelso) [inline data/101/922/189/101922189.geojson]: This is problematic. Can we set to `-1` or `XX` or something?

File: data/101/922/189/101922189.geojson [hunk 2]
```diff
@@ -44,7 +44,7 @@
     "wof:breaches":[],
     "wof:concordances":{},
     "wof:country":"PS",
```
Comment (nvkelso) [inline data/101/922/189/101922189.geojson]: This is problematic. Can we set to `-1` or `XX` or something? It should also have a dual hierarchy between Israel and Palestine?

**Comment by nvkelso** [comment]
Please also set `mz:is_current` to **1** on https://whosonfirst.mapzen.com/spelunker/id/101752329/ for Athens.

File: data/101/922/189/101922189.geojson (hunk 1 above)
Comment (stepps00) [reply data/101/922/189/101922189.geojson]: Sure, the convention here would be `XX`.

**Comment by stepps00** [comment]
Reviewer comments addressed in c6de4d1.

**Comment by stepps00** [comment]
This work is being tracked in #1283, through part of a larger task to merge duplicate records.

The logic to bring over "missing" properties in that PR is cleaner, so I'll close this PR in preference of that PR.

**Pull request closed without merging**